### PR TITLE
Add a base class for output log readability

### PR DESCRIPTION
**Pull request by maropu**
This pr makes us easily read logs output from a logger.

### CODE REVIEW

**Comment by myui** [comment]
@maropu 

I don't want to see logging messages in the results of unit tests (e.g., on  https://travis-ci.org/myui/hivemall/jobs/86948608), except debugging purposes.

So, I'm considering to set logging level to error for testing.
http://stackoverflow.com/questions/24231773/specifying-a-custom-log4j-properties-file-for-all-of-junit-tests-run-from-eclips/24315819#24315819

**Comment by myui** [comment]
Made https://github.com/myui/hivemall/pull/211 to set logging level to ERROR in tests.

File: src/test/java/hivemall/test/HivemallTest.java [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+ */
+package hivemall.test;
+
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.rules.TestName;
+
+public class HivemallTest {
```
Comment (myui) [inline src/test/java/hivemall/test/HivemallTest.java]: `abstract class HivemallTestBase` is better

File: src/test/java/hivemall/test/HivemallTest.java [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+    private static final Log logger = LogFactory.getLog(HivemallTest.class);
```
Comment (myui) [inline src/test/java/hivemall/test/HivemallTest.java]: The logging message always starts with `test.Hivemalltest`.

I think this field needs not to be static. 
`private final Log logger = LogFactor.getLog(getClass());`

File: src/test/java/hivemall/test/HivemallTest.java [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+
+    private final String packageName =
+            this.getClass().getPackage().getName();
+
+    @Rule
+    public final TestName testName = new TestName();
+
+    @Before
+    public void beforeEachTest() {
+        logger.info("\n\n===== TEST OUTPUT FOR " + packageName + ": '"
```
Comment (myui) [inline src/test/java/hivemall/test/HivemallTest.java]: logger.info("\n\n===== TEST OUTPUT FOR " + testName.getMethodName() + "' =====\n");

**Comment by maropu** [comment]
Agreed. I think developers set info log-level to see the separators for easy debugging.

File: src/test/java/hivemall/test/HivemallTest.java (hunk 3 above)
Comment (maropu) [inline src/test/java/hivemall/test/HivemallTest.java]: I think all the other hivemall tests'd be better to extend this class easily to see a relationship between tests and error messages. So, package names help to understand test names that each package has.

**Comment by myui** [comment]
LGTM. Merged. Thanks!

**Pull request merged**